lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,23 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ *
+ * @public
+ * @alias module:@ui5/builder.processors.moduleBundler~ModuleSection
+ * @typedef {object} ModuleSection
+ * @property {boolean} resolve=false Whether (transitive) dependencies of module should be resolved
```

Comment:
aren't the default values only understand when the parameter is optional (surrounded by square brackets)

Reply:
Yes you are right.